lib/git_chain/commands/delete.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+require "optparse"
+
+module GitChain
+  module Commands
+    class Delete < Command
+      include Options::ChainName
+
+      def description
+        "Delete chain"
+      end
+
+      def run(options)
+        if Git.rebase_in_progress?
+          raise(Abort, "A rebase is in progress. Please finish the rebase first and run 'git chain rebase' after.")
+        end
+
+        chain = current_chain(options)
+
+        puts_debug("Deleting chain #{chain.formatted}}}")
+
+        branches = chain.branches
+        branches.each do |b|
+          Git.set_config("branch.#{b}.chain", nil, scope: :local)
+          Git.set_config("branch.#{b}.parentBranch", nil, scope: :local)
+          Git.set_config("branch.#{b}.branchPoint", nil, scope: :local)
+        end
+  
+        unless branches.empty?
+          puts_warning("Removed #{branches.map { |b| "{{info:#{b}}}" }.join(", ")} from the chain.")
+        end
+  
```

Comment:
I considered having a `--delete` option like Prune so you can delete the chain AND delete the branches, but that seems like it might be too risky (and not useful enough to justify itself).

It could be something like this:

```
if options[:delete]
  branches.each do |branch|
    puts_info("Deleting branch {{info:#{branch}}}")
    args = ["branch", "--delete", "--force", branch]
    puts_debug_git(*args)
    Git.exec(*args)
  end
end   
```